dsp/modules/ollama.py

```diff
@@ -134,7 +142,14 @@ def basic_request(self, prompt: str, **kwargs):
             "raw_kwargs": raw_kwargs,
         }
         self.history.append(history)
-
+        if LANGFUSE:
+            langfuse.trace(
```

Comment:
this is hardcoded for "Ollama" and also tracing is hardcoded. Would it not be possible to do tracing outside of DSPy code?